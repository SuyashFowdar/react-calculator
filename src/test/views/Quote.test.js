import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../../views/Quote';

it('renders correctly', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
