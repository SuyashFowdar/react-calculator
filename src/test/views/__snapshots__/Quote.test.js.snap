// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container row cross-center h-100"
>
  <h3>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
  </h3>
</div>
`;
